products/BellHybrid/apps/application-bell-onboarding/include/application-bell-onboarding/ApplicationBellOnBoarding.hpp

```diff
@@ -43,7 +43,8 @@ namespace app
         explicit ApplicationBellOnBoarding(std::string name                    = applicationBellOnBoardingName,
                                            std::string parent                  = "",
                                            StatusIndicators statusIndicators   = StatusIndicators{},
-                                           StartInBackground startInBackground = {false});
+                                           StartInBackground startInBackground = {false},
+                                           std::uint32_t stackDepth            = 1024 * 8);
```

Comment:
how much has the application stack increased?
did you test free system heap size after changes?